docs/release.md

```diff
@@ -1,43 +1,66 @@
 # Release Guidelines
 
-## Introduction
+Welcome to Eclipse Tractus-X Release Guidelines, known as TRGs. With Release Guidelines, we are defining quality
+criteria for our repositories, products and artifacts in general. TRGs are numbered and grouped into sections, to
+provide an easy-to-use overview. The git history of the TRGs can be used to browse changes made throughout the project
+history. You can also check the [TRG Changelog](release/trg-0) to get a better view on the history of
+new/changed/deprecated TRGs.
 
-Welcome to Tractus-X Release Guidelines, known as TRGs. TRG numbers are assigned by the TRG
-editors, and once assigned never changed. The version control history of the TRG texts represent their historical
-record. The TRGs are indexed by topic for specialist subjects.
+## Process
 
-You find the overarching Changelog [here](release/trg-0) to see when new TRGs have been added.
+The process of introducing or altering a TRG is straight-forward. It consists of the following phases and events:
 
-## Process
+1. A proposal is created.
```

Comment:
The dot!! the dot!1 pls remove that dot

Reply:
[d5fcef5](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/871/commits/d5fcef5b284b43a5902d914586024321cfe2fe4d)